src/graph/service/GraphFlags.cpp

```diff
@@ -56,7 +56,7 @@ DEFINE_bool(enable_optimizer, false, "Whether to enable optimizer");
 DEFINE_uint32(ft_request_retry_times, 3, "Retry times if fulltext request failed");
 DEFINE_bool(enable_client_white_list, true, "Turn on/off the client white list.");
 DEFINE_string(client_white_list,
-              nebula::getOriginVersion() + ":2.5.0:2.5.1:2.6.0",
+              nebula::getOriginVersion() + ":3.0.0",
```

Comment:
getOriginVersion would be 3.0.0 after we release, should we add this explicit version?

Reply:
I discussed this offline with @CPWstatic and we think explicitly adding the version is clearer.